src/m3/ecore/importer.ts

```diff
@@ -0,0 +1,119 @@
+import {
+    Concept,
+    Feature,
+    FeaturesContainer,
+    Metamodel,
+    MetamodelElement,
+    PrimitiveType
+} from "../types.ts"
+import {MetamodelFactory} from "../factory.ts"
+import {hashingIdGen} from "../../id-generation.ts"
+import {
+    asArray,
+    EClassifier,
+    EcoreXml,
+    EStructuralFeature
+} from "./types.ts"
+import {ConceptType} from "../functions.ts"
+
+
+const refPrefix = "#//"
```

Comment:
Is this the prefix only for local references? Can we have references that are not local? If so, should we call it "localRefPrefix"?